src/chocolatey/infrastructure/tolerance/FaultTolerance.cs

```diff
@@ -40,7 +40,7 @@ private static bool log_is_in_debug_mode()
         }
 
         /// <summary>
-        /// Tries an action the specified number of tries, warning on each failure and erroring on the last.
+        /// Tries an action the specified number of tries, warning on each failure and raises error on the last attempt.

```

Comment:
`Erroring` definitely didn't feel right to me :-)

Perhaps `failure and raises an error on the last attempt`